include/input.h

```diff
@@ -0,0 +1,12 @@
+#ifndef INCLUDE_INPUT_H_
+#define INCLUDE_INPUT_H_
+
+/* Input struct */
+typedef struct {
+    char** commands;
+} input;
```

Comment:
Typically, to avoid name collisions, the convention is to either not `typedef` (so that you have to write `struct input` each time), or to append `_t` to the end of the type name. I don't care which you pick, but be sure to apply the change consistently.

Reply:
Let's be sure to address this. I'd prefer to have `struct input` be the way to declare these structs, please make this change.

Reply:
We actually do not use this file at all, so I'll just remove it.